packages/flutter/test/scheduler/scheduler_test.dart

```diff
@@ -116,4 +129,23 @@ void main() {
     expect(timerQueueTasks.length, 2);
     expect(taskExecuted, false);
   });
+
+  test('Flutter.Frame event fired', () async {
+    window.onReportTimings(<FrameTiming>[FrameTiming(<int>[
+      // build start, build finish
+      10000, 15000,
+      // raster start, raster finish
+      16000, 20000,
+    ])]);
+
+    final List<Map<String, dynamic>> events = scheduler.getEventsDispatched('Flutter.Frame');
+    expect(events, hasLength(1));
+
+    final Map<String, dynamic> event = events.first;
+    expect(event['number'], isNonNegative);
+    expect(event['startTime'], isNonNegative);
```

Comment:
Nit: can we make the `expect` more strict, e.g., `startTime == 10000`, `build == 5000` instead of just non-negativity? That way, we ensures that we're not mistakenly treat build times as raster times.

Reply:
👍 updated to test the results from the frame event more precisely